### PR TITLE
JAMES-3213 Change ICALToJsonAttribute replyTo header source to Reply-To

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java [hunk 1]
```diff
@@ -105,7 +105,7 @@
     public static final AttributeName DEFAULT_SOURCE = AttributeName.of(DEFAULT_SOURCE_ATTRIBUTE_NAME);
     public static final AttributeName DEFAULT_RAW_SOURCE = AttributeName.of(DEFAULT_RAW_SOURCE_ATTRIBUTE_NAME);
     public static final AttributeName DEFAULT_DESTINATION = AttributeName.of(DEFAULT_DESTINATION_ATTRIBUTE_NAME);
-    public static final String REPLY_TO_HEADER_NAME = "replyTo";
+    public static final String REPLY_TO_HEADER_NAME = "Reply-To";
```
Comment (chibenwa) [inline mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java:108]: No test broke? Do we miss a unit test?

Comment (blackheaven) [reply mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java:108]: the test uses this constant

**Comment by blackheaven** [comment]
```
[70c1886cbf678934f7948a43047f50b4ee906dbe] [ERROR]   RabbitMQDeletedMessageVaultIntegrationTest>DeletedMessageVaultIntegrationTest.vaultExportShouldExportZipContainsVaultMessagesToShareeWhenImapDeletedMailbox:543 
```

test this please

**Comment by blackheaven** [comment]
no result

test this please

**Comment by chibenwa** [comment]
no result

test this please

**Comment by chibenwa** [comment]
unrelated

test this please

File: mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java (hunk 1 above)
Comment (chibenwa) [reply mailet/icalendar/src/main/java/org/apache/james/transport/mailets/ICALToJsonAttribute.java:108]: Tests relying on production constants do have limited interest (eg this issue)

Maybe we should consider putting the raw value in tests instead.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**